front/src/styles/scss/common/components/_manchetteWithSpaceTimeChart.scss

```diff
@@ -37,6 +38,15 @@
   .manchette {
     overflow-y: auto;
     overflow-x: hidden;
+
+    &.no-scroll {
+      overflow-y: clip;
+    }
+
+    .osrd-menu {
+      width: 305px;
+      margin-top: -2px;
```

Comment:
I think it's better not to use a negative margin, a transform would be better.

Reply:
Done